docs/developer/decision-records/README.md

```diff
@@ -36,3 +36,4 @@
 - [2022-08-09 Project structure review](2022-08-09-project-structure-review/)
 - [2022-08-11 Versioning and Artifacts](2022-08-11-versioning_and_artifacts/)
 - [2022-08-17 Remove H2 Database Tests](2022-08-17-remove_h2_database_tests/)
+- [2022-09-18 IDS requests filtering](2022-09-18-ids-requests-filtering/)
```

Comment:
```suggestion
- [2022-09-18 IDS catalog request filtering](2022-09-18-ids-catalog-request-filtering/)
```
Please rename the folder according to the title.

Reply:
Done.